src/main/pg/adc.c

```diff
@@ -58,21 +58,49 @@ void pgResetFn_adcConfig(adcConfig_t *adcConfig)
 #ifdef VBAT_ADC_PIN
     adcConfig->vbat.enabled = true;
     adcConfig->vbat.ioTag = IO_TAG(VBAT_ADC_PIN);
+#if defined(STM32H7)
+#ifdef VBAT_ADC_INSTANCE
+    adcConfig->vbat.device = ADC_DEV_TO_CFG(adcDeviceByInstance(VBAT_ADC_INSTANCE));
+#else
+    adcConfig->vbat.device = adcConfig->device;
+#endif
+#endif
 #endif
 
 #ifdef EXTERNAL1_ADC_PIN
     adcConfig->external1.enabled = true;
     adcConfig->external1.ioTag = IO_TAG(EXTERNAL1_ADC_PIN);
+#if defined(STM32H7)
+#ifdef EXTERNAL1_ADC_INSTANCE
+    adcConfig->external.device = ADC_DEV_TO_CFG(adcDeviceByInstance(EXTERNAL1_ADC_INSTANCE));
```

Comment:
should this be external1.device?

Reply:
yes, this is a result of a rebase error, 2 years ago when the code first existed it was called `external` :smile: I'll update the PR.

Reply:
fixed